### PR TITLE
Change priority to use zpriority and change default priority to +inf

### PR DESCRIPTION
**Pull request by AJRenold**
For @sbarman 

cc - @yorinasub17 

Changes:
- Change priority to `zpriority` to avoid confusion with other celery priority implementations
- Change default priority to +inf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,11 @@ Kombu Transport using Redis SortedSets
...
 
     python setup.py test
+
+## Using with celery tasks - zpriority
+
+    task.apply_async(zpriority=10)
+
+zpriority is used to avoid confusion with other celery priority implementations
+
+Note - tasks created using this backend will have the lowest priority, +inf
```
Comment (sbarman) [inline README.md]: I would add "tasks created without a zpriority" just to clarify its the default

File: setup.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 setup(
```
Comment (yorinasub17) [inline setup.py:3]: You need to update the `download_url` too

File: README.md [hunk 3]
```diff
@@ -7,3 +7,11 @@ Kombu Transport using Redis SortedSets
 ## Running tests
 
     python setup.py test
+
+## Using with celery tasks - zpriority
+
+    task.apply_async(zpriority=10)
```
Comment (yorinasub17) [inline README.md]: That's pretty crazy that you can straight pass through celery a new option like this.

**Review by yorinasub17 — APPROVED** [review]
Aside from one comment about version, this looks good. A bit uncomfortable about relying on a hidden feature in celery where message properties can be set.

**Comment by sbarman** [comment]
@yorinasub17 I can see that. But I think using "priority" is worse since the ranges of priority differ and the meaning of priorities differ (in normal priority, larger means high priority). In this case, even if we switch backends, then the worst thing that happens is that we switch back to FIFO.

**Comment by yorinasub17** [comment]
@sbarman agreed that this is the better of the two evils.

**Review by sbarman — COMMENTED** [review]
Looks good overall. Did you tag this commit with the version? We will also need to push out a release to PyPi

**Pull request merged**